longhaul-test/scheduler-workflow-deploy.yml

```
@@ -0,0 +1,54 @@
+# ------------------------------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------------------------------
+
+kind: Service
+apiVersion: v1
+metadata:
+  name: scheduler-workflow
+  labels:
+    app: scheduler-workflow
+spec:
+  selector:
+    app: scheduler-workflow
+  ports:
+    - protocol: TCP
+      port: 9988
+      targetPort: 9988
+  type: ClusterIP
+
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: scheduler-workflow-app
+  labels:
+    app: scheduler-workflow
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: scheduler-workflow
+  template:
+    metadata:
+      labels:
+        app: scheduler-workflow
+      annotations:
+        dapr.io/enabled: "true"
+        dapr.io/app-id: "scheduler-workflow"
+        dapr.io/enable-profiling: "true"
+        dapr.io/log-as-json: "true"
+        prometheus.io/scrape: 'true'
+        prometheus.io/port: '9988'
+    spec:
+      containers:
+        - name: scheduler-workflow
+          image: daprtests.azurecr.io/scheduler-workflow:dev
+          ports:
+            - name: dapr
+              containerPort: 3009
```

```suggestion
              containerPort: 8484
```
Or change the port in the app file to 3009, to follow what the other apps do.

---

changed to 3009 everywhere